### PR TITLE
[SMALLFIX] Add keyvalue store related operations to JournalProtoUtils

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

**Comment by calvinjia** [comment]
LGTM, is there also a unit test to ensure these cover all the possible entries?

File: core/server/src/main/java/alluxio/master/journal/JournalProtoUtils.java [hunk 1]
```diff
@@ -67,6 +67,16 @@ public static Message unwrap(JournalEntry entry) {
         return entry.getRename();
       case SET_ATTRIBUTE:
         return entry.getSetAttribute();
+      case COMPLETE_PARTITION:
+        return entry.getCompletePartition();
+      case COMPLETE_STORE:
+        return entry.getCompleteStore();
+      case CREATE_STORE:
+        return entry.getCompleteStore();
```
Comment (calvinjia) [inline core/server/src/main/java/alluxio/master/journal/JournalProtoUtils.java]: should this be `entry.getCreateStore()`?

**Comment by apc999** [comment]
PTAL @calvinjia

**Comment by calvinjia** [comment]
LGTM thanks.

**Comment by uronce-cc** [comment]
Here is a port of this PR to master: https://github.com/Alluxio/alluxio/pull/2927

**Pull request merged**